Sentry.xcodeproj/xcshareddata/xcschemes/Sentry.xcscheme

```diff
@@ -88,11 +85,18 @@
                <Test
                   Identifier = "SentryStacktraceBuilderTests/testAsyncStacktraces_disabled()">
                </Test>
-               <Test
-                  Identifier = "SentryProfilerSwiftTests/testConcurrentSpansWithTimeout_disabled()">
-               </Test>
             </SkippedTests>
          </TestableReference>
+         <TestableReference
```

Comment:
I think this is an Xcode 14.2 thing? It's not something I added myself at least.

Reply:
I removed it by hand